src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -422,7 +420,37 @@ async Task RestartAsync()
 		}
 	}
 
-	private async Task OnMessageReceivedAsync(object? sender, NotificationRequestIdeMessage message)
+	private async Task OnMessageReceivedAsync(object? sender, IdeMessage devServerMessage)
+	{
+		try
+		{
+			switch (devServerMessage)
+			{
+				case KeepAliveIdeMessage:
+					_debugAction?.Invoke($"Keep alive from Dev Server");
+					break;
```

Comment:
As this is somehow a core feature of the `IdeChannelClient` I would have keep it's "handling" directly there.

(In a future version, instead of just logging it, we should use those to detect that channel has been broken and try to re-connect. This is something that is really internal to the `IdeChannelClient` IMO.)

Reply:
KeepAliveIdeMessage moved back to the IdeChannelClient.
https://github.com/unoplatform/uno/commit/da8f935ec4c1ea54e7e7cd4c2fbffe4841cc146f